app/controllers/concerns/lti_support.rb

```diff
@@ -144,6 +145,7 @@ def _generate_new_lti_user(params)
     user.password = ::SecureRandom::hex(15)
     user.password_confirmation = user.password
     user.lti_user_id = lti_user_id
+    user.legacy_lti_user_id = lti_user_id
```

Comment:
Canvas only sends the 1.1 lti_user_id during an OAuth request. If they change that we'd have to change it as well but if they do it would break a lot of tools so I don't know that they will change it anytime soon. I'm guessing we'll need this for years to come.